docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -0,0 +1,47 @@
+General configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+  :widths: 30, 40, 30
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default value
+  * - ``case-insensitive-name-matching``
+    - Support case insensitive database and collection names
+    - False
+  * - ``case-insensitive-name-matching.cache-ttl``
+    -
+    - 1 minute
+  * - ``metadata.cache-ttl``
+    - Duration for which metadata, including table and column statistics, is
+      cached
+    - 0 (disabled caching)
+  * - ``metadata.cache-missing``
+    - Cache the fact that metadata, including table and column statistics, is
+      not available
+    - False
+  * - ``allow-drop-table``
+    - Allow connectors to drop tables
```

Comment:
"Connectors" makes this sound like this one setting will apply to every connector. Would suggest changing to "the connector" which implies it's only applicable for this connector, without naming it.

Reply:
Pre-existing. Will change in separate PR. (I've just split the single fragment into two).